tests/InstrEngineTests/NaglerInstrumentationMethod/NaglerInstrumentationMethod.cpp

```diff
@@ -1202,10 +1202,51 @@ HRESULT CInstrumentationMethod::PerformSingleReturnInstrumentation(IMethodInfo*
     return S_OK;
 }
 
+// this function does handle multiple returns or tailcalls
```

Comment:
Do you mean "this function *does not handle*..."?

Reply:
Fixed in: https://github.com/robertpi/CLRInstrumentationEngine/commit/83506ba636b3aa43e6860b9755dca696667ddfc0